### DISCUSSION TITLE
Add varibale from a CSV file

### DISCUSSION THREAD

**Discussion by Hernon7** [grafana/grafana-csv-datasource#25] [Q&A] [answered]
Hello and thanks for your contribution.
I would like to know how to create a variable from one specific column using a csv data source. 
![image](https://user-images.githubusercontent.com/44420775/106230946-8b975a80-622b-11eb-917c-ad30ce4b21af.png)

If use a normal SQL database, the query would be:
`select your_col_name from your_table`

Thanks agian and looking for your response🙏

**Comment by marcusolsson** [comment, marked as answer]
Unfortunately the plugin doesn't yet support variable queries. If this is important to you, feel free to submit a feature request :)

**Reply by Hernon7** [reply]
Thanks for your quick reply, I'll definitely consider submit the request
